Finder/TranslationFinder.php

```diff
@@ -14,14 +14,14 @@ class TranslationFinder
     /**
      * @var KernelInterface
      */
-    protected $kernel;
+    protected $translationFilesByLocale;
```

Comment:
I suggest making the new property private, to avoid having it covered by any backward compatibility concern